third_party/requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Python requirements modules
 # Formatting the source code
 clang-format==15.0.7
+cmake-format==0.6
```

Comment:
what happens if I use different version? is it compatible or not necessarily, like in clang-format

Reply:
That's a good question. I think that if there is some required version, it should be stated in the README file, just like for cleng-format. Otherwise, drop the exact version from this file.